.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
do you intend to commit this?

Reply:
yes. otherwise, it is easy to commit this folder unintentionally.